app/src/main/res/menu/bottom_nav_menu.xml

```diff
@@ -1,19 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
-
+    <!-- icons subject to change  -->
     <item
-        android:id="@+id/navigation_home"
-        android:icon="@drawable/ic_home_black_24dp"
-        android:title="@string/title_home" />
+        android:id="@+id/navigation_library"
+        android:icon="@drawable/ic_home_black_24"
```

Comment:
Maybe change the name of all the drawable icons to match the corresponding fragment name? Everything else looks good for me

Reply:
imo icons might be used more than one place, might not even be a fragment who's using it. It should then keep its name independent and only describe what it looks like. i.e. home icon is named home because it looks like a home instead of where it's used.

Reply:
agreed with lucas